CGPs/cgp-0103

```diff
@@ -0,0 +1,30 @@
+---
+cgp: 103
+title: Dunia/Ecolabs launching eXOF on CELO
+date-created: 2023-9-29
+author: "Serge Kiema <sergek@dunia.africa>, Serge Ouedraogo <serge_o@dunia.africa>"
+Forum Post Discusssion: https://forum.celo.org/t/dunia-launching-exof-on-celo/6261/1
+
+---
+
+## Overview
+
+eXOF Launch Governance proposal
+
+This proposal covers the last phase for the launch of eXOF. With this launch eXOF become the first African currency available on Celo. The goal is to onboard the next wave of local crypto user through a familiar and user friendly stablecoin. With our launch partners we are making eXOF Available through OTC , mobile money and cash point across the XOF zone.
+A more detailed description of the motivation behind this launch can be found here (https://docs.google.com/document/d/1kb_UmcOP6VuAUMUpn_emP86qjS5I48m-et6mj24VrgM/edit?usp=sharing) . An in-depth description of the governance proposal aimed at approvers and developers can be found in the forum post.
```

Comment:
This could be added here as text and not a link for historical purposes.

Reply:
thanks @0xzoz. This has been fixed.